soh/soh/Enhancements/randomizer/3drando/settings.cpp

```diff
@@ -2561,6 +2561,9 @@ namespace Settings {
 
     MapsAndCompasses.SetSelectedIndex(cvarSettings[RSK_STARTING_MAPS_COMPASSES]);
 
+    BombchusInLogic.SetSelectedIndex(cvarSettings[RSK_BOMBCHUS_IN_LOGIC]);
+    AmmoDrops.SetSelectedIndex(AMMODROPS_VANILLA); // Ensure logic knows bombchu drops aren't implemented yet
```

Comment:
now that chu drops are implemented https://github.com/HarbourMasters/Shipwright/pull/1257 we should figure out how best to handle this

i know @aMannus has done some work on forcing/disabling settings in imgui, this feels like a perfect opportunity to implement a setting that is forced based on something in the seed/save file

Reply:
Bombchu drops are a seperate option in 3DS rando, and are also seperately considered in logic. We will have to include the bombchu drops in the randomizer menu (which then forces the vanilla enhancement to be on), but I'm not sure if that has to happen in this PR. Especially because I'd rather wait for the PR moving ImGui out of LUS to do the enhancement menu changes required.

So basically, we either have this PR on hold until the ImGui cleanup is done/merged in and we add the bombchu drops in this PR, or we merge this in first and add the bombchu drops part in a seperate PR.

Reply:
i'm fine with that plan

@lilDavid if you resolve the merge conflicts on this PR and add a comment in `z_en_girla` explaining the early return is to make sure bombchus don't go out of stock this can be good to go

then we'll want to make an issue for bombchu drops in logic